rbc/src/main/resources/application.yml

```diff
@@ -0,0 +1,10 @@
+spring:
+  application:
+    name: rbc
+  data:
+    mongodb:
+      host: localhost
+      port: 27017
+      database: library
+      username: library
+      password: library
```

Comment:
Suggestion:
Use `spring.data.mongodb.uri: mongodb://<username>:<password>@<host>:<port>/<database>` property instead of individual properties. MongoDB connection strings (URIs) can be easily reused between different environments and tools.